worksheet_D/oxo.py

```diff
@@ -1,26 +1,50 @@
 class OxoBoard:
     def __init__(self):
         """ The initialiser. Initialise any fields you need here. """
-        raise NotImplementedError("TODO: implement __init__")
+        self.gameboard = [0, 0, 0, 0, 0, 0, 0, 0, 0]
```

Comment:
There are neater ways to initialise the list, e.g. with a loop or using list multiplication